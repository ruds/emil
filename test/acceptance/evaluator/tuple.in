();
(2);
(2, "43");
(it, (it, "foo"));
(14i : int, "43") : (int * string);
