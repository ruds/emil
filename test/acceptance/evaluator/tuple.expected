val it = () : ()
val it = 2 : bigint
val it = (2, "43") : (bigint * string)
val it = ((2, "43"), ((2, "43"), "foo")) : ((bigint * string) * ((bigint * string) * string))
val it = (14i, "43") : (int * string)
